src/ontology/foodon-edit.owl

```diff
@@ -57823,12 +57838,17 @@ SubClassOf(obo:FOODON_03460172 obo:FOODON_00002454)
 # Class: obo:FOODON_03460173 (food (salted))
 
 AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 obo:IAO_0000428)
+AnnotationAssertion(obo:IAO_0000114 obo:FOODON_03460173 "requires curation"@en)
+AnnotationAssertion(obo:IAO_0000115 obo:FOODON_03460173 "A food product which is output of a salting process where salt is dissolved or absorbed into the food as an ingredient or flavourant."@en)
 AnnotationAssertion(obo:IAO_0000412 obo:FOODON_03460173 "http://langual.org")
+AnnotationAssertion(dc:contributor obo:FOODON_03460173 "https://orcid.org/0000-0001-5193-0062 | https://orcid.org/0000-0001-8008-8249"@en)
 AnnotationAssertion(oboInOwl:hasDbXref obo:FOODON_03460173 "http://www.langual.org/langual_thesaurus.asp?termid=H0173")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:FOODON_03460173 "salted"@en)
+AnnotationAssertion(oboInOwl:hasSynonym obo:FOODON_03460173 "food (compounded)"@en)
 AnnotationAssertion(rdfs:comment obo:FOODON_03460173 "LanguaL curation note: Used when it is necessary to distinguish one product from another (e.g., salted vs. unsalted peanuts), or for salt-fermented products (e.g. soy sauce). It can also be used for other products known to contain at least 4% added and naturally present salt (sodium chloride).
 When salting is used as preservation method, index *preservation by SALTING [J0103]* or narrower term."@en)
 AnnotationAssertion(rdfs:label obo:FOODON_03460173 "food (salted)"@en)
+AnnotationAssertion(rdfs:seeAlso obo:FOODON_03460173 "https://www.lawinsider.com/dictionary/compound-food#:~:text=compound%20food%20means%20food%20which%20contains%20food%20grade,undesirable%20effect%20on%20the%20characteristics%20of%20such%20food%3B"@en)
```

Comment:
It looks to me like the URL's ":~" stuff isn't needed and could be dropped.  And similarly for straight URL entries, use the "IRI editor" field - that sets it up to be recognized as a URL for other RDF systems.

Reply:
Thanks for the feedback, will check the other URLs too. Have amended in later commit.